test/api/createJobTests.js

```diff
@@ -0,0 +1,39 @@
+const proxyquire = require('proxyquire');
+const sinon = require("sinon");
+const chai = require("chai");
+const assert = chai.assert;
+
+const auth = {
+    url: "test",
+    user: "user",
+    password: "password"
+};
+
+const jobArgs = {
+        repository: "test",
+        branch: "master",
+        name: "test"
+    }
+
+const dataMock = {name:"Test"};
+
+const JenkinsWrapper = {
+    job: {
+        create: sinon.stub().yields(null, dataMock)
+    }
+};
+
+const jenkinsStub = () =>
+    JenkinsWrapper;
+
+const createJob = proxyquire("../../lib/api/createJob", { 'jenkins': jenkinsStub });
+
+describe('createJob', function () {
+    it('it should return job', function () {
+        createJob(auth, jobArgs, function (err, data) {
+            assert.isNull(err, "Should not return error")
```

Comment:
Could it happen that the error is not null but `undefined`? Should we maybe use `isNotOk()` instead?